pkg/apis/troubleshoot/v1beta2/collector_shared.go

```diff
@@ -123,6 +123,16 @@ type Database struct {
 	URI           string `json:"uri" yaml:"uri"`
 }
 
+type RRD struct {
+	CollectorMeta   `json:",inline" yaml:",inline"`
+	Namespace       string            `json:"namespace" yaml:"namespace"`
+	Image           string            `json:"image" yaml:"image"`
+	ImagePullPolicy string            `json:"imagePullPolicy,omitempty" yaml:"imagePullPolicy,omitempty"`
+	ImagePullSecret *ImagePullSecrets `json:"imagePullSecret,omitempty" yaml:"imagePullSecret,omitempty"`
+	Timeout         string            `json:"timeout,omitempty" yaml:"timeout,omitempty"`
+	HostPath        string            `json:"hostPath" yaml:"hostPath"`
```

Comment:
We can default to the kURL well-known path if this is empty, right?

Reply:
Yes, this will be injected by kots and added to our default spec.